2-mandatory/2-currency-code-lookup.js

```diff
@@ -18,7 +18,11 @@ const COUNTRY_CURRENCY_CODES = [
 ];
 
 function createLookup(countryCurrencyCodes) {
-  // write code here
+  let currencyCodesObject = {};
+  countryCurrencyCodes.forEach(element => {
+    currencyCodesObject[element[0]] = element[1]    
+  });
+  return currencyCodesObject;
 }
 
```

Comment:
I have used the forEach method to loop though the aray, the solution used a for of loop